app/(candidate)/dashboard/upgrade-to-pro/components/UpdateToProPage.js

```diff
@@ -52,11 +79,24 @@ export default function DetailsPage(props) {
             is reinvested into making it possible for more people across the
             country to run, win and serve.
           </Body2>
-          <Link href="/dashboard/pro-sign-up">
-            <PrimaryButton className="flex items-center justify-center mx-auto mb-16 w-full md:w-auto">
+          <Link onClick={handleLinkOnClick} href="/dashboard/pro-sign-up">
```

Comment:
The user can still see the link when they hover over the link.
Wouldn't it be better to go with
```
<Link href={demoPersona ? '#' : '/dashboard/pro-sign/up'}>
```

Reply:
Also, if a demo user goes directly to `/dashboard/pro-sign-up` will they be able to upgrade? we need to make sure we block that.

Reply:
https://github.com/thegoodparty/gp-webapp/pull/244/commits/cac7da553149567db8c8aabfa2994981cdc039d6